sw/host/opentitanlib/src/spiflash/flash.rs

```diff
@@ -55,6 +55,7 @@ impl Default for SpiFlash {
 impl SpiFlash {
     // Well known SPI Flash opcodes.
     pub const READ: u8 = 0x03;
+    pub const FAST_READ_4B: u8 = 0x0B;
```

Comment:
Opcode 0x0B is just "FAST_READ", not "FAST_READ_4B" (that opcode is 0x0C).

It works correctly for us because prior to issuing any reads, `SpiFlash` issues `ENTER_4B`, thus causing the eeprom to want a 4-byte address for opcode 0x0B.

Reply:
Done